docker/Dockerfile

```diff
@@ -77,26 +65,8 @@ RUN groupadd --gid 1000 odc \
   && install -d -o odc -g odc /code \
   && true
 
-COPY --from=env_builder /bin/tini /bin/tini
-USER odc
-RUN --mount=type=bind,from=env_builder,target=/b \
-    --mount=type=cache,target=/home/odc/.cache/pip,uid=1000,gid=1000 \
-    env-build-tool new_no_index /b/conf/requirements.txt /b/conf/constraints.txt /env /b/wheels
-
-# All steps from above should cache well
-# from here down things depend on entire content of code
-RUN --mount=type=bind,from=env_builder,target=/b \
-    --mount=type=bind,target=/src \
-    --mount=type=cache,target=/home/odc/.cache/pip,uid=1000,gid=1000 \
-    (cd /src && tar c . ) | (cd /code && tar x) \
-    && /env/bin/python -m pip install -e '/code/[all]' --no-index --find-links=/b/wheels \
-    && /env/bin/python -m pip install -e /code/examples/io_plugin --no-index --find-links=/b/wheels \
-    && /env/bin/python -m pip install -e /code/tests/drivers/fail_drivers --no-index --find-links=/b/wheels
-
 USER root
-COPY docker/assets/with_bootstrap /usr/local/bin/
-
 VOLUME /code
 WORKDIR /code
 
-ENTRYPOINT ["/bin/tini", "-s", "--", "/usr/local/bin/with_bootstrap"]
+ENTRYPOINT ["/bin/tini", "-s", "--", "/usr/local/bin/with_bootstrap"]
```

Comment:
Are we still doing this `with_bootstrap` thing?

Not sure we need to use `tini`

Reply:
It seems to work, so I guess this is fine.

Reply:
Honestly, I don't fully understand what tini is giving us.  We are running a postgres database and the datacube in the one docker image, so maybe it lets that run smoother.

Reply:
https://jupyter-notebook.readthedocs.io/en/stable/public_server.html#docker-cmd